ush/hpssarch_gen.sh

```diff
@@ -282,7 +282,49 @@ if [ $type = "gfs" ]; then
 fi   ##end of gfs
 #-----------------------------------------------------
 
+if [ $ATMTYPE = "DATA" ]; then
+  if [ $DO_OCN = "YES" ]; then
```

Comment:
Is this just a duplicate of the other `DO_OCN`, etc. blocks for GFS? Instead of duplicating, can we restructure so those sections are used in both cases?